homeassistant/components/config/core.py

```diff
@@ -83,6 +84,7 @@ async def websocket_update_config(
     try:
         await hass.config.async_update(**data)
         if update_units:
+            async_update_number_units(hass)
```

Comment:
This seems to be unrelated to the topic of the PR. Please make a separate PR if this needs changing.